tpv12_13/parameters.par

```diff
@@ -64,9 +63,8 @@ ClusteredLTS = 2                               ! 1 for Global time stepping, 2,3
 /
 
 &Output
-FaultOutputFlag = 1                            ! DR output (add this line only if DR is active)
 OutputFile = 'output/tpv13'
-Format = 10                                    ! Format (10= no output, 6=hdf5 output)
+wavefieldoutput = 10                                    ! Format (10= no output, 6=hdf5 output)
```

Comment:
Would be best to have `wavefieldoutput=1` here (`10` will work as well, but it's less clear what it means then IMO)

Reply:
> Also, the other examples could also need such updates. :)

Agreed. I would do the others sometime later. I wanted to quickly update this as I keep using this example for quick testing everywhere.